lib/search_click_tracking/response.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'common/client/concerns/service_status'
+require 'common/models/base'
+
+module SearchClickTracking
```

Comment:
Can this response class be removed? Doesn't look like it's being used currently.

Reply:
Possibly! I will get it a test.

Reply:
Removed it!